src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/V2FeedPackageInfo.cs

```diff
@@ -230,8 +231,10 @@ public IReadOnlyList<PackageDependencyGroup> DependencySets
                             {
                                 var versionRangeString = parts[1];
 
+                                // Nexus will write "null" when there is no depenency version range.
                                 // Parse the optional version range
-                                if (!string.IsNullOrEmpty(versionRangeString))
+                                if (!string.IsNullOrWhiteSpace(versionRangeString) 
+                                    && !string.Equals(NullString, versionRangeString, StringComparison.Ordinal))
```

Comment:
No, just ordinal here. If we really want to ignore everything else we can do a VersionRange.TryParse and ignore anything invalid, but the fix here is just to support nulls for nexus.